contracts/plugins/assets/curve/CurveStableMetapoolCollateral.sol

```diff
@@ -172,13 +169,6 @@ contract CurveStableMetapoolCollateral is CurveStableCollateral {
         // Add-in contribution from pairedTok
         // {UoA} = {UoA} + {UoA/pairedTok} * {pairedTok}
         aumLow += lowPaired.mul(pairedBal, FLOOR);
-
-        // Add-in high part carefully
-        uint192 toAdd = highPaired.safeMul(pairedBal, CEIL);
-        if (aumHigh + uint256(toAdd) >= FIX_MAX) {
-            aumHigh = FIX_MAX;
-        } else {
-            aumHigh += toAdd;
-        }
+        aumHigh += highPaired.mul(pairedBal, CEIL);
```

Comment:
So, do we _want_ to revert if it overflows? I feel like the existing implementation here would be better for safeguarding the return. ~Or~ And should be using `safeMul`, I think.

Reply:
nvm

Reply:
Noting for others: we want to revert here because otherwise it's possible for low to be non-zero while high is FIX_MAX. Reverting here causes `tryPrice()` to revert which is what we want.